lambdas/process-journey-event/src/main/java/uk/gov/di/ipv/core/processjourneyevent/ProcessJourneyEventHandler.java

```diff
@@ -167,6 +167,12 @@ public Map<String, Object> handleRequest(JourneyRequest journeyRequest, Context
                         auditEventUser, stepResponse.getMitigationStart(), deviceInformation);
             }
 
+            String journeyAuditEvent = stepResponse.getAuditEvent();
+            if (journeyAuditEvent != null) {
+                AuditEventTypes auditEventType = AuditEventTypes.valueOf(journeyAuditEvent);
```

Comment:
Hmm will this blow up if the event name doesn't exist in the enum? we probably want to handle this with a nice error